pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -605,6 +605,19 @@ self: super: {
     '';
   });
 
+  d-bus = let
+    # Unreleased version for reasons...
```

Comment:
Can you please specify those reasons, e.g. by linking to an issue?

Reply:
I have created an issue for this and added its link to the file.